### ISSUE TITLE
Google Code should point to this repo

### ISSUE THREAD

**Issue by erikcw** [clintecker/django-google-analytics#1]
I just discovered this repo.  I've been using the google code version for awhile.  The Google code homepage should be updated telling people to look here for the latest code...

**Comment by doismellburning** [comment]
Agreed. It's still the top hit for "google analytics django" (how I found it) - took a remarkable amount of digging to find this (and associated pypi entry)
